xml/depl_maintenance.xml

```diff
@@ -508,6 +508,262 @@
    </itemizedlist>
   </sect2>
 
+  <sect2 xml:id="sec.depl.maintenance.postgre-mariadb.upgrade">
+   <title>Preparing &postgres;-Based &productname; 7 for Upgrade</title>
+   <para>
+    Upgrading &productname; 7 is only possible when it uses &mariadb; deployed
+    with the Database &barcl;. This means that before you can proceed with
+    upgrading &productname; 7, you must migrate &postgres; to &mariadb; first. The following description covers several possible scenarios.
+   </para>
+   <sect3 xml:id="sec.postgre-mariadb.upgrade.scenario1">
+    <title>Non-HA setup or HA Setup with a 2-Node Cluster and
+    &postgres; Database Backend</title>
+    <para>
+     Before you proceed, make sure that the database cluster has an odd number
+     of nodes. If necessary, extend the cluster as described in <xref linkend="sec.postgre-mariadb.upgrade.scenario2"/>. Install the latest maintenance updates on &productname; 7. In the
+     &crow; &wi;, switch to the <guimenu>Database</guimenu> &barcl;. You should
```

Comment:
I believe there are actually 2 sections describing extending the cluster (one with DRBD one without).

(It might be better to merge them into one and make the DRBD part just an extra note in such section)

Reply:
This is a bit confusing. Could you elaborate on that, please?

Reply:
Ah, got it! Fixed.